day3/README.md

```diff
@@ -20,3 +26,93 @@ Start the app:
+Now that we are able to fire an event with an action creator, we need to be able to pick up this event, this is where the reducers come in.
+We want all the cards to flip when we press reset. Currently, our cards are static data, we need to move them into the `reducers.js` file and make them respond to the action we have dispatched.
+Thus, we copy the card data from `data.js` and set the initial card state like so 
+```
+const initialStateCards = [
+                              { id: 0, item: 'http://lorempixel.com/200/200/cats/1/', matched: false, open: false},
+                              { id: 1,item: 'http://lorempixel.com/200/200/cats/6/', matched: true, open: true},
+                              { id: 2,item: 'http://lorempixel.com/200/200/cats/4/', matched: false, open: false},
+                              { id: 3,item: 'http://lorempixel.com/200/200/cats/5/', matched: true, open: true},
+                              { id: 4,item: 'http://lorempixel.com/200/200/cats/2/', matched: false, open: false},
+                              { id: 5,item: 'http://lorempixel.com/200/200/cats/2/', matched: false, open: false},
+                              { id: 6,item: 'http://lorempixel.com/200/200/cats/1/', matched: false, open: true},
+                              { id: 7,item: 'http://lorempixel.com/200/200/cats/4/', matched: false, open: true},
+                              { id: 8,item: 'http://lorempixel.com/200/200/cats/3/', matched: false, open: false},
+                              { id: 9,item: 'http://lorempixel.com/200/200/cats/6/', matched: true, open: true},
+                              { id: 10,item: 'http://lorempixel.com/200/200/cats/3/', matched: false, open: false},
+                              { id: 11,item: 'http://lorempixel.com/200/200/cats/5/', matched: true, open: true},
+                          ]
+```
+
+Now, inside the reducer function named `cardReducer` there is a switch statement waiting for the action `GAME_RESET`. 
+In here is where you should reset the cards.
+You want to map over the array of cards (`state.map`) and set `open` to `false` and `matched` to `false`.
+You can check if you are doing things correctly by inspecting the `cardReducer` object in the Redux panel and see if the cards have changed.
+ 
+Now, to complete this task, we need to insert the state data into our `App`, instead of the static data from `data.js`. 
+The `App` is already connected to the store by the statement at the bottom of the file:
+```
+export default connect(state => ({
+    cards: state.cardReducer
+}))(App);
+```
+Thus, all you need to do is to pass `props.cards` to the `Game` component. Now, when you press the Restart button, all cards should flip!
+    
+### Task 3: Flip a single card - practicing the Redux pattern 
+Start by creating a new action creator `flipCard(cardData)` that returns the type `CARD_FLIP` in `actions.js`. Remember to export the type at the top of the file.
```

Comment:
Skjønner. Se forøvrig https://github.com/bekk/funksjonell-frontend/pull/9#discussion_r57567887.